.travis.yml

```diff
@@ -2,6 +2,7 @@
 language: generic
 
 os: osx
+osx_image: xcode6.1
```

Comment:
Also raised an upstream issue about erroring out builds where `osx_image` refers to a non-existent image.

xref: https://github.com/travis-ci/travis-ci/issues/6684